lib/mixins/properties-mixin.html

```diff
@@ -87,10 +88,14 @@
      * @return {Object} Memoized properties object
      */
     function ownProperties(constructor) {
-      if (!constructor.hasOwnProperty(
-        JSCompiler_renameProperty('__ownProperties', constructor))) {
-        const props = constructor.properties;
-        constructor.__ownProperties = props ? normalizeProperties(props) : null;
+      if (!constructor.hasOwnProperty(JSCompiler_renameProperty('__ownProperties', constructor))) {
+        let props = null;
+
+        if (constructor.hasOwnProperty('properties') && constructor.properties) {
```

Comment:
I imagine we probably need `JSCompiler_renameProperty` for `properties`, don't we?

Reply:
The weird part is, with or without produces the same output. Nonetheless, I have added the line.